cli/command/formatter/disk_usage.go

```diff
@@ -37,36 +37,61 @@ type DiskUsageContext struct {
 	Volumes     []*types.Volume
 	BuildCache  []*types.BuildCache
 	BuilderSize int64
+
+	trunc bool
 }
 
 func (ctx *DiskUsageContext) startSubsection(format string) (*template.Template, error) {
 	ctx.buffer = bytes.NewBufferString("")
 	ctx.header = ""
-	ctx.Format = Format(format)
+	ctx.Context.Format = Format(format)
 	ctx.preFormat()
 
 	return ctx.parseFormat()
 }
 
-// NewDiskUsageFormat returns a format for rendering an DiskUsageContext
-func NewDiskUsageFormat(source string) Format {
-	switch source {
-	case TableFormatKey:
-		format := defaultDiskUsageTableFormat
-		return Format(format)
-	case RawFormatKey:
-		format := `type: {{.Type}}
+// Format sets a format for rendering an DiskUsageContext
+func (ctx *DiskUsageContext) Format(source string) {
```

Comment:
I did this to access ctx.Verbose, but I think I'll revert this to the function to keep the change minimal and just add `verbose` param instead.